apps/nowcasting-app/constant.ts

```diff
@@ -102,3 +112,5 @@ export enum DELTA_BUCKET {
   POS4 = 100
 }
 export const getDeltaBucketKeys = () => Object.keys(DELTA_BUCKET).filter((k) => isNaN(Number(k)));
+
+export const NHR_FORECAST_OPTIONS = [1, 2, 4, 8];
```

Comment:
There's a few place where `Nhr` and NHour` get used, Should we try to set them as the same? Perhaps use `NHour` as its slightly more explicit.

Reply:
Have refactored, hopefully almost all out now – left ENV vars just for the moment as can add later